Libplanet.Explorer/Queries/ExplorerQuery.cs

```diff
@@ -17,9 +16,11 @@ namespace Libplanet.Explorer.Queries
 {
     public class ExplorerQuery : ObjectGraphType
     {
+        private static IBlockChainContext? chainContext;
```

Comment:
Maybe use `_chainContext` to avoid name collision in `ExplorerQuery()`? We generally use Pascal case or `_` prefixed camel case for `private static`.